tests/unit/s2n_security_policies_test.c

```diff
@@ -590,28 +628,13 @@ int main(int argc, char **argv)
     {
         struct s2n_config *config = s2n_config_new();
 
-        EXPECT_SUCCESS(s2n_config_set_cipher_preferences(config, "default"));
-        EXPECT_EQUAL(config->security_policy, &security_policy_20170210);
-        EXPECT_EQUAL(config->security_policy->cipher_preferences, &cipher_preferences_20170210);
-        EXPECT_EQUAL(config->security_policy->kem_preferences, &kem_preferences_null);
-        EXPECT_EQUAL(config->security_policy->signature_preferences, &s2n_signature_preferences_20140601);
-        EXPECT_EQUAL(config->security_policy->ecc_preferences, &s2n_ecc_preferences_20140601);
```

Comment:
I'm removing these tests because I'm not sure what value they actually add. They just redefine the policies.